packages/tests/generated-packages/project.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "tests-generated-packages",
+  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "packages/tests/generated-packages/src",
+  "projectType": "library",
+  "targets": {
+    "lint": {
+      "executor": "@nx/linter:eslint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": ["packages/tests/generated-packages/**/*.ts"]
+      }
+    },
+    "test": {
+      "executor": "@nx/jest:jest",
+      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
+      "dependsOn": ["build-all-no-playgrounds"],
```

Comment:
@Lp-Francois How can I make this `dependsOn` actually work? 
Currently it doesn't seem to invoke the `build-all-no-playgrounds` target.

Reply:
Answered on Slack